README.md

```diff
@@ -1,15 +1,65 @@
 [![Travis](https://api.travis-ci.org/bids-standard/bids-examples.svg?branch=master "Travis")](https://travis-ci.org/bids-standard/bids-examples)
 
-# BIDS-examples
-A set of BIDS compatible datasets with **empty raw data files** that can be used for writing lightweight software tests.
+# bids-examples
+
+This repository contains a set of BIDS compatible datasets with **empty raw
+data files**. These datasets can be useful to:
+
+1. write lightweight software tests
+1. serve as an example on how a BIDS dataset can be structured
 
 ALL RAW DATA FILES IN THIS REPOSITORY ARE EMPTY!
```

Comment:
It might be worth noting that, for the synthetic dataset, the NIfTI files are truncated at their headers so they can be loaded and their metadata read.

Reply:
good point :+1: the header is also intact for some EEG examples. I'll add this.